test/system/030-run.bats

```diff
@@ -821,4 +821,30 @@ EOF
     run_podman run --rm $IMAGE cat /proc/self/oom_score_adj
     is "$output" "$current_oom_score_adj" "different oom_score_adj in the container"
 }
+
+# CVE-2022-1227 : podman top joins container mount NS and uses nsenter from image
+@test "podman top does not use nsenter from image" {
+    tmpdir=$PODMAN_TMPDIR/build-test
+    mkdir -p $tmpdir
+    tmpbuilddir=$tmpdir/build
+    mkdir -p $tmpbuilddir
+    dockerfile=$tmpbuilddir/Dockerfile
+    cat >$dockerfile <<EOF
+FROM $IMAGE
+RUN rm /usr/bin/nsenter
+RUN echo -e "#!/bin/sh\n\
+false" \
+>> /usr/bin/nsenter
+RUN chmod +x /usr/bin/nsenter
```

Comment:
This should be all done in one RUN statement to save some time.

Reply:
done